[RLlib] Cleanup metrics-spam in algo's result dicts (retire "sampler_results", don't dump env-runner-results into top-level). (#45378)
ray-project/ray: doc/source/rllib/rllib-cli.md

> Let's run the example next!

Reviewer: ```suggestion
Run the example next.
```

> After showing how to start the training run, we give you some sample output of it below.

Reviewer: ```suggestion
After following the instructions for starting the training run, see some sample output below.
```

> Note that by default, RLlib will create an indicative experiment name for you, and logs

Reviewer: ```suggestion
Note that by default, RLlib creates an indicative experiment name for you, and logs
```
